src/main/java/main_yuta.java

```diff
@@ -0,0 +1,166 @@
+import com.sun.xml.internal.bind.v2.TODO;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.revwalk.RevCommit;
+import org.refactoringminer.api.GitService;
+import org.refactoringminer.util.GitServiceImpl;
+
+import satd_detector.core.utils.SATDDetector;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class main_yuta {
+    public static int true_number = 0;
+    public static int false_number = 0;
+    public static List<String> true_message = new ArrayList<>();
+
+    public static List<String> all_true = new ArrayList<>();
+
+    public static List<String> false_message = new ArrayList<>();
+    public static void main(String[] args) throws IOException, InterruptedException {
+        Path directory = Paths.get("C:\\Users\\wm124\\summer_inter\\satd1\\aspectj.eclipse.jdt.core");
+        GitService gitService = new GitServiceImpl();
+
+
+        //TODO jgit
+        try{
+            Repository repository = gitService.openRepository("aspectj.eclipse.jdt.core");
```

Comment:
ここ上のdirectoryと同じパスを指していると思うので
```suggestion
            Repository repository = gitService.openRepository(directory.toString());
```
が動くならそのほうが良い